lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -0,0 +1,114 @@
+var edx = edx || {};
+
+(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortDiscussionsView = Backbone.View.extend({
+
+        /**
+         Add/Remove the disabled attribute on given element.
+
+         Args:
+            $element (JQuery element): The element to disable/enable.
+            disable (Bool): The flag to add/remove 'disabled' attribute.
+        **/
```

Comment:
Minor nit: this isn't a valid JSDoc comment. It needs to look like these:

http://usejsdoc.org/about-getting-started.html

Each line starts with ``* `` and then the final line ends with ``*/``.

We aren't yet generating documentation from our JavaScript, but I hope we do relatively soon.